Source/FuzzySearch.swift

```diff
@@ -48,11 +68,37 @@ public struct FuzzySearchResult {
 
 public protocol FuzzySearchable {
     var fuzzyStringToMatch: String { get }
+    
+    func fuzzyTokenized() -> FuzzyTokens
```

Comment:
Shouldn't this be a private method in private `FuzzySearchable` extension? Is it worth leaving a possibility to override it?